scripts/mbedtls_framework/config_common.py

```diff
@@ -412,6 +433,14 @@ def main(self):
                 if value:
                     sys.stdout.write(value + '\n')
             return 0 if args.symbol in config else 1
+        elif args.command == 'get-all':
+            match_list = config.get_matching(args.regexs, False)
+            if match_list is not None:
```

Comment:
Please remove the `is not None` tests, they're no longer needed.

Reply:
Ops sorry, I forgot to remove those lines while doing the previous changes.